docs/docs/setting-up-ci-cd.mdx

```diff
@@ -36,112 +37,73 @@ resource-intensive tests only when certain files have been changed or when some
-data, or story data. To validate your data, have your CI run this command:
-
-```bash
-rasa data validate
-```
-
-If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the 
-smallest of those values as
-
-```bash
-rasa data validate --max-history <max_history>
-```
-
-If data validation results in errors, training a model can also fail or yield bad performance, so it's
-always good to run this check before training a model. By including the
-`--fail-on-warnings` flag, this step will fail on warnings indicating more minor issues.
-
-:::note
-Running `rasa data validate` does **not** test if your [rules](./rules.mdx) are consistent with your stories. 
-However, during training, the `RulePolicy` checks for conflicts between rules and stories. Any such conflict will abort training.
-:::
-
-To read more about the validator and all of the available options, see [the documentation for 
-`rasa data validate`](./command-line-interface.mdx#rasa-data-validate).
-
-### Training a Model
-
-You can train a model with the following command:
-
-```bash
-rasa train
-```
-
-Training a model verifies that your NLU pipeline and policy configurations are
-valid and trainable, and provides a model to use for testing your assistant.
-If the model passes the CI tests, then you can also [upload the trained model](./setting-up-ci-cd.mdx#deploying-your-rasa-model)
-to your server as part of the continuous deployment process.
-
-
-### Testing Your Assistant
-
-Testing your trained model on [test stories](./testing-your-assistant.mdx#writing-test-stories) is the best way to have confidence in how your assistant
-will act in certain situations. Written in a modified story
-format, test stories allow you to provide entire conversations and test that, given certain
-user input, your model will behave in the expected manner. This is especially
-important as you start introducing more complicated stories from user
-conversations.
-
-The command to run test stories is:
-
-```bash
-rasa test --fail-on-prediction-errors
+### CI Pipeline Overview
+
+There are different approaches to applying a CI pipeline depending on the number of changes.
+All follow a mix of training the new model, testing to compare to older versions,
+and setting up an easy CD pipeline to upload the new model.
+
+The first step after saving new training data is to kick off the pipeline. (Is it usually kicked off
+manually or triggered by an event like saving a new file?)
+(need different pipelines / options based on how big the changes are - also should we lay out how
+small or big of changes warrant which tests?) Typically this is done by xyz
+
+Next, you need to pick what level of tests to run based on the amount of data changed. The various types
+are laid out in [Testing Your Assistant](https://rasa.com/docs/rasa/testing-your-assistant), but a good
+rule of thumb is to always run your test stories to ensure no bugs were introduced. Larger changes
```

Comment:
This can be re-worded to mention running data validation, NLU cross validation, and story testing, and then referring to the Testing Your Assistant page if they want more details